modules/command-engine/javadsl/src/main/scala/surge/javadsl/command/SurgeCommand.scala

```diff
@@ -84,6 +84,10 @@ private[javadsl] class SurgeCommandImpl[AggId, Agg, Command, Evt](
     FutureConverters.toJava(healthCheck().map(_.asJava))
   }
 
+  def getReady: CompletionStage[HealthCheck] = {
```

Comment:
Maybe `getReadiness` for naming on this?

Reply:
done